PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
could you move Breaking change to the top of this list? As if a user does not know what a breaking change is, it will be useful to explain it first before using the term in the other items